src/app/project/project.component.ts

```diff
@@ -199,10 +199,7 @@ export class ProjectComponent implements OnInit {
   }
 
   canSubmit(project: Project) {
-    return this.userIsWrangler &&
-      !project.hasOpenSubmission &&
-      project.validationState.toUpperCase().replace(' ', "_") !== SUBMISSION_STATES.GraphValid &&
-      !(project.validationErrors && project.validationErrors.length > 0);
+    return this.userIsWrangler && !project.hasOpenSubmission;
```

Comment:
Why remove all of the logic? It's quite nice not to submit it no?

Reply:
This is per @gabsie's request. We now want to allow users to update the project upon spreadsheet upload. We want to let users upload a submission even if the project is invalid.